misc/systemd/ipfs.service

```diff
@@ -3,9 +3,39 @@ Description=InterPlanetary File System (IPFS) daemon
 After=network.target
 
 [Service]
+ReadWritePaths="/var/lib/ipfs/"
+NoNewPrivileges=true
+ProtectSystem=strict
+ProtectKernelTunables=true
+ProtectKernelModules=true
+ProtectKernelLogs=true
+PrivateDevices=true
+DevicePolicy=closed
+ProtectControlGroups=true
+RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6 AF_NETLINK
+ProtectHostname=true
+PrivateTmp=true
```

Comment:
This breaks the mount feature, unfortunately.

Reply:
Yeah, I haven't yet tested out all features. That's why it's still a draft.

Basic HTTP-Gateway functionality with QUIC and URLStore/Filestore works fine - since Filestore is limited to /var/lib/ipfs anyway.